keyboards/nazare160/beta/rules.mk

```diff
@@ -0,0 +1,26 @@
+MCU = STM32F411
```

Comment:
```suggestion
# MCU name
MCU = STM32F411
```

Reply:
Committed in [e2895d8](https://github.com/qmk/qmk_firmware/pull/16822/commits/e2895d81e81b944b3f008b0ffa48599be08d54bd)